charts/umbrella/docs/contributing.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
```

Comment:
I recommend integrating the contents of this file with the one on root level, to have all the relevant information one file.

Reply:
Done with the new commit in a view minutes ;)